infinitic-engine/src/main/kotlin/io/infinitic/engine/workflowManager/engines/ForWorkflowTaskEngine.kt

```diff
@@ -0,0 +1,45 @@
+package io.infinitic.engine.workflowManager.engines
+
+import io.infinitic.common.tasks.messages.ForTaskEngineMessage
+import io.infinitic.common.tasks.messages.TaskCompleted
+import io.infinitic.common.workflows.data.methodRuns.MethodRunId
+import io.infinitic.common.workflows.data.workflowTasks.WorkflowTask
+import io.infinitic.common.workflows.data.workflowTasks.WorkflowTaskId
+import io.infinitic.common.workflows.data.workflowTasks.WorkflowTaskOutput
+import io.infinitic.common.workflows.data.workflows.WorkflowId
+import io.infinitic.common.workflows.messages.TaskCompleted as ForWorkflowTaskCompleted
+import io.infinitic.common.workflows.messages.WorkflowTaskCompleted
+import io.infinitic.engine.taskManager.engines.TaskEngine
+import io.infinitic.engine.taskManager.storage.TaskStateStorage
+import io.infinitic.messaging.api.dispatcher.Dispatcher
+
+class ForWorkflowTaskEngine(
+    override val storage: TaskStateStorage,
+    override val dispatcher: Dispatcher
```

Comment:
```suggestion
    storage: TaskStateStorage,
    dispatcher: Dispatcher
```

Storage and dispatcher are already defined as properties in the parent class. By making them protected in the parent class they will become accessible in this class so I believe you can remove all modifiers to make them constructor parameters and it should work.